cpl/module_cap_cpl.F90

```diff
@@ -13,9 +17,17 @@ module module_cap_cpl
   public diagnose_cplFields
   contains
 
-  !-----------------------------------------------------------------------------
-  !-----------------------------------------------------------------------------
-
+    !> ???
+    !> 
+    !> @param[in] gcomp ???
```

Comment:
@junwang-noaa can you please tell us what the ??? should be in this documentation?